test/test_test_app.rb

```diff
@@ -19,37 +19,49 @@ def fixture_path(*args)
 
 class TestTestApp < Minitest::Test
   def test_resources_pod_returns_spec_path
-    assert_nil(resources_pod(Pathname.new('/')))
-    assert_nil(resources_pod(Pathname.new('.')))
+    %i[ios macos].each do |target|
```

Comment:
Sometimes it may be preferable to move enumeration of test variants outside of the test and generate multiple tests instead, so you have a better understanding of what failed when an assertion fails. E.g.:

```ruby
class TestTestApp < Minitest::Test
  %i[ios macos].each do |target|
    define_method("test_#{target}_resources_pod_returns_spec_path") do
      assert_nil(resources_pod(Pathname.new('/'), target))
      # ...
    end
  end
end
```

Reply:
I guess you could also create `Minitest::Test` subclasses in a loop instead, unsure off-hand what’s more helpful with minitest.

Reply:
Thanks, I'll go with the first suggestion for now to keep it simple.